features/vaadin-dashboard/src/main/java/org/opennms/features/vaadin/dashboard/ui/HeaderLayout.java

```diff
@@ -59,17 +65,19 @@ public HeaderLayout() {
          * Setting up the layout
          */
         addStyleName("header");
-        setMargin(true);
+        setMargin(new MarginInfo(false,true,false,false));
         setSpacing(true);
         setWidth("100%");
+        setHeight(64, Unit.PIXELS);
 
         /**
          * Adding the logo
          */
         Link link = new Link(null, new ExternalResource("/opennms/index.jsp"));
-        link.setIcon(new ThemeResource("img/logo.png"));
+        link.setIcon(new ExternalResource("/opennms/images/horizon_logo.svg"));
```

Comment:
Can you set the icon based on css? 
This is definetly causing trouble for a Meridian release.